utils/types.go

```diff
@@ -94,6 +94,11 @@ type KongClientConfig struct {
 	HTTPClient *http.Client
 
 	Timeout int
+
+	SessionFilePath string
+
+	// Whether to initialize the Http client with a cookie jar or not
+	ISSessionClient bool
```

Comment:
Is this required? Why not check if the SessionFilePath != ""?

Reply:
Yeah, I could do that, felt that the boolean could satisfy multiple flags, so left it. I can remove it

Reply:
Done